File: Examples/Features/Formatting/Alignment/WrapText.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.Alignment
{
    public class WrapText : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            IRange rangeB3 = worksheet.Range["B3"];
            rangeB3.Value = "The WrapText property is applied to wrap the text within a cell";
            rangeB3.WrapText = true;

            worksheet.Rows[2].RowHeight = 150;
        }
    }
}

File: Examples.VB/Features/Formatting/Alignment/WrapText.vb
Namespace Features.Formatting.Alignment
    Public Class WrapText
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            Dim rangeB3 As IRange = worksheet.Range!B3
            rangeB3.Value = "The WrapText property is applied to wrap the text within a cell"
            rangeB3.WrapText = True
            worksheet.Rows(2).RowHeight = 150
        End Sub
    End Class
End Namespace
